config/7.8.0/common.yaml

```diff
@@ -268,6 +268,11 @@ console:
                     mountPath: "[[.Console.GitHooks.MountPath]]"
                     readOnly: true
                   #[[end]]
+                  #[[if .Console.GitHooks.SSHSecret]]
+                  - name: "[[.Constants.GitHooksSSHSecret]]"
+                    mountPath: "/home/jboss/.ssh"
+                    readOnly: true
```

Comment:
my only concern about it is the permissions,
for directory it should be 700 and files 644.

Reply:
I've fixed the perms issue... but i'm also concerned about overwriting the user's .ssh dir. Is there no way the product can look in another dir for ssh keys so we can leave the ~/.ssh dir alone?
@mramendi ?

Reply:
@tchughesiv I have checked the 7.7.0 production BC image during out call. A /home/jboss/.ssh directory does not exist there, so there is nothing to overwrite.

Reply:
For safety, perhaps the directory itself should not be read-only? In case the product writes there at runtime?